engine/apps/alerts/constants.py

```diff
@@ -22,3 +23,27 @@ class AlertGroupState(str, Enum):
     ACKNOWLEDGED = "acknowledged"
     RESOLVED = "resolved"
     SILENCED = "silenced"
+
+
+# ServiceNow Integration
```

Comment:
Wondering if these constants could be defined somewhere else? They seem very specific for ServiceNow